Front-end/parquick/src/components/Map/Map.jsx

```diff
@@ -96,9 +118,21 @@ function Map({ isLoaded }: Props): React.MixedElement {
 
     const markerLoadHandler = (marker, place) => {
         return setMarkerFMap(prevState => {
-            return { ...prevState, [place.id]: marker };
+            return { ...prevState, [place._id]: marker };
         });
     };
+    const markerHoverHandler = async (event, place) =>{
+        await setInfoIsOpen(false);
+        await setSelectedPlace(null);
+        await setInfoIsOpen(true);
+        await setSelectedPlace(place);
+        //await setIsJustOnHover(true);
```

Comment:
nit: remove commented out code

also I think we don't need `await` in all of these as they are useState setters?